src/pages/Home/components/PopupMarker/style.scss

```diff
@@ -76,8 +76,7 @@
 
   button:hover {
     background-color: #508991;
-    font-weight: 800;
-    cursor: pointer;
```

Comment:
is it `cursor: pointer` by default then ?

Reply:
Yes.